server/scripts/migrations.py

```diff
@@ -0,0 +1,50 @@
+import json
+import re
+
+import click
+from girder.models.file import File
+from girder.models.folder import Folder
+from girder.models.item import Item
+from girder.models.user import User
+
+from dive_server import crud, crud_annotation
+
+
+@click.command(name="migrate")
+@click.option('--dry-run', is_flag=True, default=False)
+def migrate_server(dry_run):
+    admin_user = User().findOne({'admin': True})
+    click.echo(f'Running as user {admin_user["login"]}')
+    total_tracks_ingested = 0
+    annotation_files = Item().find(
+        {'meta.detection': {'$exists': True}, 'meta.migrated': {'$exists': False}},
+        sort=[['created', 1]],
+    )
+    for annotation in annotation_files:
+        if annotation['name'].endswith('.json'):
+            dataset_id = annotation["meta"]["detection"]
+            dataset = Folder().load(dataset_id, user=admin_user)
+            crud.verify_dataset(dataset)
```

Comment:
Before this you may want to validate that the folder is a valid annotation folder? (meta.annotate:True).  Or you may want to handle the instance where the `verify_dataset` fails without killing the entire loop.  Maybe outputting at the end of the loop a list of datasets that couldn't be migrated?